public/linkedhashes.php

```diff
@@ -41,7 +41,7 @@
              "<a href='https://docs.retroachievements.org/Game-Identification/'>here</a>." .
              "</b></p>";
 
-        echo "<p class='mt-4 mb-1'>There are currently <span class='font-bold'>" . count($hashes) . "</span> supported game file hashes registered for this game.</p>";
+        echo "<p class='mt-4 mb-1'>There " . (count($hashes) === 1 ? "is" : "are") . " currently <span class='font-bold'>" . count($hashes) . "</span> supported game file " . Str::plural('hash', count($hashes)) . " registered for this game.</p>";
```

Comment:
Recommend calculating `count($hashes)` once.

Reply:
Addressed in https://github.com/RetroAchievements/RAWeb/pull/1431/commits/d9d96adac1040ceeb5cce766f2205b91e09e6b26. I have also changed `Str::plural()` to instead use `__res()`.